colorable_windows.go

```diff
@@ -430,8 +431,14 @@ func atoiWithDefault(s string, def int) (int, error) {
 	return strconv.Atoi(s)
 }
 
+var (
+	writeMutex sync.Mutex
```

Comment:
This mutex have to be a field of Writer.

Reply:
Thanks, fixed